packages/graph/hash_graph/lib/graph/src/store/postgres/database_type.rs

```diff
@@ -1,54 +1,32 @@
-use crate::types::{
-    schema::{DataType, EntityType, LinkType, PropertyType},
-    VersionedUri,
-};
+use crate::ontology::types::{DataType, EntityType, LinkType, OntologyType, PropertyType};
 
 /// Provides an abstraction over elements of the Type System stored in the Database.
 ///
-/// [`PostgresDatabase`]: crate::datastore::PostgresDatabase
-pub trait DatabaseType {
-    /// Returns the unique versioned URI used to identify this instance of a type.
-    fn uri(&self) -> &VersionedUri;
-
+/// [`PostgresDatabase`]: crate::store::PostgresDatabase
+pub trait DatabaseType: OntologyType {
```

Comment:
Do we require the bound? I.e. **must** `DatabaseType` also implement `OntologyType`?